docs/pt/docs/tutorial/request-forms.md

```diff
@@ -0,0 +1,58 @@
+# Dados do formulário
+
+Quando você precisa receber campos de formulário ao invés de JSON, você pode usar `Form`.
```

Comment:
I think `Quando você precisar...` in the infinitive sounds better

Reply:
Done.

Reply:
You can remove the first *você*, and add an *r* at the end of *precisa*. This way, you will avoid repeating words and keeping the infinitive in the phrase